Tests/UnitTests/FoundationExtensions/NSData+RCExtensionsTests.swift

```diff
@@ -29,14 +29,29 @@ class NSDataExtensionsTests: TestCase {
     }
 
     func testAsFetchToken() {
-        let receiptFilename = "base64EncodedReceiptSampleForDataExtension"
-        let storedReceiptText = NSDataExtensionsTests.readFile(named: receiptFilename)
-        let storedReceiptData = NSDataExtensionsTests.sampleReceiptData(receiptName: receiptFilename)
+        let storedReceiptText = NSDataExtensionsTests.readFile(named: Self.receiptFilename)
+        let storedReceiptData = NSDataExtensionsTests.sampleReceiptData(receiptName: Self.receiptFilename)
         let fetchToken = storedReceiptData.asFetchToken
 
         expect(fetchToken).to(equal(storedReceiptText))
         expect(storedReceiptData.asFetchToken).to(equal(storedReceiptText))
     }
+
+    func testStringDataAsUUID() {
+        expect("sample string".asData.uuid) == UUID(uuidString: "73616D70-6C65-2073-7472-696E67000000")
+    }
+
+    func testDataAsUUIDHashesTheEntireData() {
```

Comment:
Hold on this new test fails ⚠️ 
I need to use an actual hashing algorithm. 

I luckily realized this was wrong by looking into the implementation of this method: https://opensource.apple.com/source/CF/CF-855.17/CFUUID.c.auto.html